CHANGES.md

```diff
@@ -1,13 +1,18 @@
 # Changes
 
+## 0.5.0 (2025-04-??)

```

Comment:
I left the exact date blank, so that you can fill it out when you're ready for a new release.